notebooks/rxrx3_core_benchmarks_openphenom.ipynb

```diff
@@ -0,0 +1,2858 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    metadata_columns
</pre><p>This line can be cleaned</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/recursionpharma/EFAAR_benchmarking/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>